### ISSUE TITLE
/mnt does not show dynamically-appearing drive letters E: (CD/DVD)

### ISSUE THREAD

**Issue by jamesdavidhoward** [Microsoft/WSL#560]
If a DVD or USB stick is inserted into the PC, Windows dynamically sees the media content, and (if needed) assigns or re-uses a drive letter. Example: DVD inserted visible as **`E:\`**...  However, the DVD is not visible as a **`/mnt`**" point like other Windows 10 NTFS drives.  Since "**`/mnt`**" is an artificially-constructed entity, there is no adequate reason that "**`/mnt/e`**" cannot be as correspondingly visible/present as "**`/mnt/c`**"

You've defined a simple, meaningful, bi-directionally-understandable translation for the Windows and WSL environments.  Keep it that way.

**Comment by dethoma** [comment]
Right now WSL only supports fixed NTFS/ReFS (with limitations) volumes. We don't monitor for removable drives today.

**Comment by benhillis** [comment]
There is a [thread on our user voice page](https://wpdev.uservoice.com/forums/266908-command-prompt-console-bash-on-ubuntu-on-windo/suggestions/13355724-unable-to-access-usb-devices-from-bash) requesting this support.  It's fairly high on the list of requested features and definitely something that is on our radar.

**Comment by jamesdavidhoward** [comment]
Thanks for your referral to the User Voice page for bash.  However, I am
familiar with this web area.

Because there are so many locations established or in-use to provide bug
reports and/or feedback and/or feature requests, I have NO concrete idea
where to submit such suggestions.  I strongly suspect others have a
similar problem.  You would do well to unify these disparate locations
into one.
___
James D. Howard, jhoward@alumni.caltech.edu

**Comment by benhillis** [comment]
Agreed, we're trying to find a single platform that supports all of our needs and have yet to find one that does issue report and feature requests equally well.  You are welcome to submit suggestions to either place.  We appreciate your feedback.

**Comment by benhillis** [comment]
Closing out old issues. Support for mounting filesystems on removable drives (USB, CD, etc) was added in Fall Creators Update.

**Comment by therealkenc** [comment]
@benhillis - this one is being used (in some amorphous unofficial capacity) as the landing zone auto-mount/auto-unmount like [message](https://github.com/Microsoft/WSL/issues/1926#issuecomment-294324183). It's a place to dupe things down when they come in.

**Comment by benhillis** [comment]
Fair enough.

**Comment by therealkenc** [comment]
Made it even. :)
